edge/pkg/devicetwin/dtmanager/membership.go

```diff
@@ -1,20 +1,20 @@
 package dtmanager
 
 import (
-	"github.com/kubeedge/kubeedge/beehive/pkg/common/log"
-	"github.com/kubeedge/kubeedge/beehive/pkg/core"
-	"github.com/kubeedge/kubeedge/beehive/pkg/core/model"
+	"github.com/kubeedge/kubeedge/common/beehive/pkg/common/log"
+	"github.com/kubeedge/kubeedge/common/beehive/pkg/core"
+	"github.com/kubeedge/kubeedge/common/beehive/pkg/core/model"
```

Comment:
please move the github.com/* section together with the following lines that importing github.com/*